about.html

```diff
@@ -48,7 +52,7 @@ <h3>"Hello! Pets lovers"</h3>
             <h1>Veterinary international meet alternative natural food</h1>
         </div>
         
-        <p>The meeting to present an alternative diet for our pets is held 
+        <p id="introDesc">The meeting to present an alternative diet for our pets is held 
             annually to present the advances in this field. A more balanced 
             and natural diet represents an improvement in the quality of life 
             and a reduction in visits to the clinic for our pets.</p>
```

Comment:
- 🙏Please decrease the size of the box🎁 containing the description and change the background color of the box to white, matching the original design. This adjustment will help preserve the coolness🎉 of your design, similar to the original.

Your design:
![image](https://github.com/Dachrono/capstone/assets/63445083/01221710-50fe-4e0c-8e3a-2a621a989d59)
Original design:
![image](https://github.com/Dachrono/capstone/assets/63445083/a65c5192-7014-4c3f-81b4-d0bac1099712)

Reply:
Solved, tky for you advise